devito/arch/compiler.py

```diff
@@ -410,9 +412,13 @@ def __init__(self, *args, **kwargs):
                 self.ldflags += ['-march=%s' % platform.march]
         elif platform is M1:
             # NOTE:
-            # -march=native unsupported
-            # openmp unusable
-            pass
+            # Apple M1 supports OpenMP through Apple's LLVM compiler.
+            # The compiler can be installed with Homebrew or can be built from scratch.
+            # Check if installed and set compiler flags accordingly
+            llvmm1 = get_m1_llvm_path()
```

Comment:
I would switch to llvm_m1 for readability
or....m1_llvm for more homogeneity with the get_m1...function